### PR TITLE
updated old version numbers that got missed by release process

### PR DESCRIPTION
**Pull request by zippy**
## PR summary

It appears that sometimes old version numbers didn't get updated by the release process, probably if stuff was stuck in branches that were worked on over a long time and then merged poorly.

For this PR, I did a manual search for all version numbers from "0.0.7-alpha" to "0.0.17-alpha" and fixed them to be 0.0.18-alpha1 which is what's current at the time of this branch.  Thus this should get merged before next release so the scripts don't miss these updates again!

## testing/benchmarking notes

( if any manual testing or benchmarking was/should be done, add notes and/or screenshots here )

## followups

( any new tickets/concerns that were discovered or created during this work but aren't in scope for review here )

## changelog

Please check one of the following, relating to the [CHANGELOG-UNRELEASED.md](https://github.com/holochain/holochain-rust/blob/develop/CHANGELOG-UNRELEASED.md)

- [ ] this is a code change that effects some consumer (e.g. zome developers) of holochain core so it is added to the CHANGELOG-UNRELEASED.md (linked above), with the format `- summary of change [PR#1234](https://github.com/holochain/holochain-rust/pull/1234)`
- [ ] this is not a code change, or doesn't effect anyone outside holochain core development

### CODE REVIEW

File: doc/holochain_101/src/first_steps.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # First steps writing Holochain hApps with Rust
 
 ___
-This tutorial builds for the 0.0.9-alpha release but as the API and HDK are changing it will likely fail under newer releases.
+This tutorial builds for the 0.0.18-alpha1 release but as the API and HDK are changing it will likely fail under newer releases.
```
Comment (Connoropolous) [inline doc/holochain_101/src/first_steps.md]: @zippy this one can't be updated unless the tutorial is actually updated too, which I think someone is doing that open source in another PR

**Comment by Connoropolous** [comment]
@zippy yah we would prbly also want to make the relevant changes to holonix release scripts to make sure it happens ongoing

**Comment by zippy** [comment]
I think holonix looks for the "previous" value, but they got skipped some times.  I did add a note about this in the release process.

File: doc/holochain_101/src/first_steps.md (hunk 1 above)
Comment (jamesray1) [reply doc/holochain_101/src/first_steps.md]: Agreed. What I'll do is I'll update to the latest release and check that the tutorial works fine.

**Pull request merged**